### PR TITLE
Updated detail of embedded application

### PR DESCRIPTION
**Pull request by metodej**
- added name and mail of user to application detail for embedded application and removed empty form

### CODE REVIEW

**Review by Vojtech-Sassmann — APPROVED** [review]

File: perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java [hunk 1]
```diff
@@ -115,10 +116,26 @@ public Widget draw() {
...
 		vp.setCellHeight(sp, "100%");
 
-		GetApplicationDataById data = new GetApplicationDataById(applicationId);
-		data.setShowAdminItems(false);
-		data.retrieveData();
-		sp.setWidget(data.getContents());
+		if (application.getType().equalsIgnoreCase("EMBEDDED")) {
+			final FlexTable ft = new FlexTable();
+			ft.setCellSpacing(5);
+			ft.setHTML(0, 0, "<strong>Name:</strong> " + SafeHtmlUtils.fromString(application.getUser().getFullNameWithTitles()).asString());
```
Comment (balcirakpeter) [inline perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java:122]: Couldn't there be a problem when there is no name and titles?

File: perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java [hunk 2]
```diff
@@ -115,10 +116,26 @@ public Widget draw() {
...
+			new GetEntityById(PerunEntity.RICH_USER_WITH_ATTRS, application.getUser().getId(), new JsonCallbackEvents(){
+				@Override
+				public void onFinished(JavaScriptObject jso) {
+					User user = jso.cast();
+					String mail = SafeHtmlUtils.fromString(user.getAttribute("urn:perun:user:attribute-def:def:preferredMail").getValue()).asString();
```
Comment (balcirakpeter) [inline perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java:127]: Same here. What will happen when the user does not have a preferred mail set?

File: perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java (hunk 1 above)
Comment (metodej) [reply perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java]: I haven't thought of that, I have added a check.

File: perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java (hunk 2 above)
Comment (metodej) [reply perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java]: I have added check here as well, thanks.

**Review by balcirakpeter — APPROVED** [review]

**Pull request merged**